spec/ruby/core/string/shared/dedup.rb

```diff
@@ -0,0 +1,47 @@
+describe :string_dedup, shared: true do
+  it 'returns self if the String is frozen' do
+    input  = 'foo'.freeze
+    output = input.send(:dedup)
```

Comment:
Should be `.send(@method)` (same below)

Reply:
🤦

Reply:
Sorry,I had some kind of brainfart. Fixed now.